.travis.yml

```diff
@@ -4,18 +4,35 @@ arch:
   - ppc64le
 
 rvm:
-  - 2.5.0
-  - 2.5.8
-  - 2.6.0
-  - 2.6.10
-  - 2.7.0
-  - 2.7.6
-  - 3.0.1
+  - 2.7.7
+  - 2.7.8
   - 3.0.4
-  - 3.1.0
+  - 3.0.6
   - 3.1.2
+  - 3.1.4
+  - 3.2.2
   - ruby-head
   - jruby-head
+
+jobs:
+  exclude:
+  # ref: https://rubies.travis-ci.org/index.txt
```

Comment:
Latest version aren't supported for `ppc64le`.
ref: https://rubies.travis-ci.org/index.txt

Reply:
It looks like they are. Where are you seeing that they are not supported?

TravisCI show all of the Rubies that we are testing. https://rubies.travis-ci.org/ They support Ruby 3.1.2 on `ppc64le`. I don't think that 3.2 was released when I last updated the tested Ruby versions. We should include those newer versions in testing.

I removed Ruby 2.4 form testing because it was having build issues on Travis. Though, IIRC, it would still build and test locally in my VMs.